cmd/infra/aws/util/util.go

```diff
@@ -31,6 +33,50 @@ func NewSession(agent string, credentialsFile string, credKey string, credSecret
 	return awsSession
 }
 
+func NewStsSession(agent, credentialsFile, roleArn, region string) (*session.Session, error) {
```

Comment:
Looks like `NewSession()` calls `session.Must()` - either refactor `NewSession()` to return an error or follow its lead and panic in `NewStsSession()` - having parallel constructs that have similar DX helps a lot.